app/assets/stylesheets/govuk-component/_title.scss

```diff
@@ -1,16 +1,18 @@
-.govuk-title {
+// scss-lint:disable SelectorFormat
+.pub-c-title {
   @include responsive-vertical-margins;
 
-  .context {
-    @include core-24;
-    color: $secondary-text-colour;
+  &--long .pub-c-title__text {
```

Comment:
We try to avoid nesting if it's not required see: https://github.com/alphagov/govuk-frontend/blob/master/docs/coding-standards/css.md#nesting